src/main/java/com/pinterest/secor/parser/DateMessageParser.java

```diff
@@ -52,7 +52,27 @@ public DateMessageParser(SecorConfig config) {
         String result[] = { defaultDate };
 
         if (jsonObject != null) {
-            Object fieldValue = jsonObject.get(mConfig.getMessageTimestampName());
+            String messageTimestampName = mConfig.getMessageTimestampName();
```

Comment:
This code is exactly the same as the one in JsonMessageParser, can they be consolidated in one place?  e.g. in MessageParser.java?